engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -207,11 +214,13 @@ class WorkflowManagerActor(config: Config)
       data.workflows.get(id) foreach {_ ! SubscribeTransitionCallBack(sender())}
       stay()
     case Event(RestartWorkflows(w :: ws), _) =>
-      val updatedData = restartWorkflow(w)
+      val updatedData = restartWorkflow(w) onSuccess {
```

Comment:
The `val` here is not used.

`restartWorkflow` used to return the full state data, now it just returns a new entry to be added.  The name of the variables in the `case` should be updated accordingly, they're currently misleading/wrong.